geopandas/base.py

```diff
@@ -143,13 +183,68 @@ def type(self):
 
     @property
     def length(self):
-        """Returns a ``Series`` containing the length of each geometry."""
+        """Returns a ``Series`` containing the length of each geometry.
+
```

Comment:
Sidenote: should we clarify what "length" means eg in case of a polygon?  (I suppose perimeter?)

Reply:
I'll go through text parts of docstrings later in different PR. There are more in a need of clarification.

Reply:
My comment above re: CRS (and implied one here) could go into that other PR instead of here.